client/my-sites/plugins/plugin-navbar-header/index.jsx

```diff
@@ -0,0 +1,662 @@
+import './style.scss';
+import { useTranslate } from 'i18n-calypso';
+
+const PluginNavbarHeader = () => {
+	const translate = useTranslate();
+
+	return (
+		<div>
+			<div className="x-root lpc-header-nav-wrapper">
+				<div className="lpc-header-nav-container">
+					{ /*<!-- Nav bar starts here. -->*/ }
+					<div className="masterbar-menu">
+						<div className="masterbar">
+							<nav className="x-nav" aria-label={ translate( ' WordPress.com' ) }>
+								<ul className="x-nav-list x-nav-list--left">
+									<li className="x-nav-item">
+										<a
+											role="menuitem"
+											className="x-nav-link x-nav-link--logo x-link"
+											href="//wordpress.com/"
+										>
+											<svg
+												xmlns="http://www.w3.org/2000/svg"
+												viewBox="0 0 170 36"
+												className="x-icon x-icon--logo"
+											>
```

Comment:
For this WordPress.com logo, we already have the SVG at several places:
- client/components/wordpress-wordmark
- client/landing/browsehappy/logo-wide.jsx
Let's reuse one of the existing ones.

And maybe there's a way to lazy load the SVG, like it was an image? @sgomes or @flootr worked on lazy loading for Gridicons, they might remember how it works.

Reply:
Thanks, @jsnajdr! Yes, it's definitely best to avoid SVG-in-JS; JS bytes are much much heavier for the browser to handle, so in general anything that **can** be kept off of JS **should** be kept off of JS. 

If using an `<img>` is sufficient here, all you need to do is add the SVG file to the appropriate resource folder, import it, and use the import value as the source name:

```jsx
import svgLogo from '...';

const logo = <img src={ svgLogo } alt="Plugin logo" />;
```

If you need to style it, then `<img>` is not an option. You'll need to create a small inline SVG, and use SVG's `<use>` to reference the external resource. The icon component source should be a good reference for how to do this.

Reply:
> If you need to style it, then `<img>` is not an option.

I think we might need to style it, so that the logo matches the text color. Logged-out masterbars are probably always white-on-black, but custom themes are sometimes dark-text-on-light-background, like the "Light" theme:

<img width="346" alt="Screenshot 2022-09-26 at 12 17 59" src="https://user-images.githubusercontent.com/664258/192252630-b1b20535-d27a-408a-b1da-f920f4c82d6c.png">

Reply:
If that's the case, perhaps this code can follow the approach in `/client/reader/sidebar/reader-sidebar-organizations/list.jsx` and reuse the generic `components/svg-icon` component:

```jsx
import AutomatticLogo from 'calypso/assets/images/icons/a8c-logo.svg';

[...]

  return <SVGIcon name="a8c-logo" icon={ AutomatticLogo } classes="sidebar__menu-icon" />;
```